doc/INSTALL.md

```diff
@@ -66,7 +66,6 @@ Add the path where the library has been installed
 
 and run `ldconfig` as root.
 
-
```

Comment:
Yes, at the moment "Installation Manuals" sounds like a good idea, we can also add src/plugins/mozprefs/autoconfig/README.md there.